hw3/AilurusUmbra/test_Matrix.py

```diff
@@ -0,0 +1,96 @@
+from _matrix import Matrix
+from _matrix import multiply_naive, multiply_mkl
+import pytest
+import numpy as np
+import time
+
+def get_rand_float(low=0, high=10, decimal=1):
+  return np.random.randint(low, high, size=(1,1)).astype(float)
+
+def get_rand_dim(low=300, high=500):
+  m = np.random.randint(low, high)
+  n = np.random.randint(low, high)
+  k = np.random.randint(low, high)
+  return m, n, k
+
+def compareMat(mat1, mat2, m, n):
+  for i in range(m):
+    for j in range(n):
+      if mat1[i,j]!= mat2[i,j]:
+        print(f'm1[i,j]:{mat1[i,j]}')
+        return False
+  return True
+
+# ============
+# === Test ===
+# ============
+
+def test_constructor():
+  m, n, _ = get_rand_dim()
+  m1 = Matrix(m, n)
+  m2 = Matrix(m, n)
+  for i in range(m):
+    for j in range(n):
+      m1[i,j] = 666.666
```

Comment:
it's better to use random number here.
How do you know if m1[i,j] actually is the same with m2[i,j]? Maybe m1[i,j] is equal to m2[i-1, j-1] if your code implementation is wrong.

Reply:
Thanks, I modify these lines to make sure two matrix should be different.